readthedocs/core/templatetags/privacy_tags.py

```diff
@@ -20,15 +20,6 @@ def is_member(user, project):
     return AdminPermission.is_member(user, project)
 
 
-@register.filter
-def projects_admin(user, privacy_level=None):
```

Comment:
Should we adapt this code to be able to easily be passed into the email?

Reply:
We used this filter to email people about the privacy level migration, for the email reminder we can just use this function content as `context_function`.